datanode/gateway/graphql/schema.graphql

```diff
@@ -5293,7 +5293,7 @@ type Reward {
   "The asset this reward is paid in"
   asset: Asset!
   "The market ID for which this reward is paid if any"
-  marketId: ID!
+  marketId: ID! @deprecated(reason: "Use gameId")
```

Comment:
Could the "game ID" field have a better name that reflects what it provides when there isn't a game, for example?

This is definitely an improvement on labeling it market ID though!